src/redux/data.ts

```diff
@@ -57,17 +57,37 @@ export interface IncomeW2 {
 }
 
 export enum Income1099Type {
-  INT = 'INT'
+  B = 'B',
+  INT = 'INT',
+  DIV = 'DIV'
 }
 
 export const form1099Types: Income1099Type[] = [
-  Income1099Type.INT
+  Income1099Type.B,
+  Income1099Type.INT,
+  Income1099Type.DIV
 ]
 
-export interface Income1099 {
-  payer: string
+export interface BData {
+  shortTermProceeds: number
+  shortTermCostBasis: number
+  longTermProceeds: number
+  longTermCostBasis: number
+}
+
+export interface IntData {
   income: number
-  formType: Income1099Type
+}
+
+export interface DivData {
+  dividends: number
+  qualifiedDividends: number
+}
+
```

Comment:
I think these should be 1099BData, 1099IntData, 1099DivData for clarity

Reply:
Done